examples/recovery/simple.json

```diff
@@ -0,0 +1,21 @@
+{
+  "ver": "1.3.0",
+  "nam": {
+    "fn": "Smith-Jones",
+    "fnt": "SMITH<JONES",
+    "gn": "Charles Edward",
+    "gnt": "CHARLES<EDWARD"
+  },
+  "dob": "1964",
+  "r": [
+    {
+      "tg": "840539006",
+      "fr": "2021-06-01",
+      "co": "NL",
+      "is": "Ministry of Health Welfare and Sport",
+      "df": "2021-06-12",
+      "du": "2021-11-28",
+      "ci": "urn:uvci:01:NL:dadfcc47c7334e45a906db12fd859fb7"
```

Comment:
UVCI should be in uppercase letters

Reply:
Good spot, thanks! I based these examples off of submissions found in the TestData repository.